AjobK/Athena
# Can now view all routes documented via swagger

closes #292 

<br />

 **What is it supposed to do**  
When NODE_ENV is set to 'development', a documentation page is hosted on /swagger. This displays all routes with their HTTP method and path. After executing different routes either with swagger or just regularly in the application, the documentation supplies itself with more information.

<br />

 **How can we test your branch**  
Run the backend after having set NODE_ENV to 'development' (in the .env file). Go to localhost:8000/swagger, the page should be there.
Run the backend after having set NODE_ENV to 'production' (in the .env file). Go to localhost:8000/swagger, the page should NOT be there.

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist** not relevant

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers** not relevant

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select at least two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.
